src/components/Navbar/NavigationDrawer/NavigationDrawerBodySkeleton.module.scss

```diff
@@ -0,0 +1,16 @@
+.skeletonContainer {
+  display: flex;
+  flex-direction: column;
+}
+
+.navRow {
+  width: auto;
+  height: 40px;
+  margin: 8px 10px 4px;
```

Comment:
A couple of things: 
1. We don't use hardcoded values for the height, we use the spacing tokens from the theme file. 
2. The margin should use logical css instead to properly support rtl layouts.